content/posts/newsletter/2019-12-03.md

```diff
@@ -0,0 +1,124 @@
+---
+title: XMPP in all languages! 03 Dec 2019
+save_as: 2019/12/newsletter-03-december/index.html
+url: 2019/12/newsletter-03-december/
+date: 2019-12-03 15:00
+author: nyco
+blog_id: newsletter
+category: Newsletter
+---
+
+## Intro 
+
+Welcome to the XMPP newsletter covering the month of November 2019.
+
+Help us sustain this as a community effort, of which [process is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+
+## Articles 
+
+Edivaldo Brito has written in Portugese "[Como instalar o moderno cliente Jabber/XMPP Dino no Linux via Snap](https://www.edivaldobrito.com.br/cliente-jabber-xmpp-dino-no-linux/)" ("How to install the modern Jabber/XMPP Dino client on Linux via Snap").
+
+![Dino](https://framapic.org/OewIIui3hyiP/UiRW9IhRFoBF.png)
```

Comment:
I'm not sure relying on external services to host pictures is a great idea. Plus Framasoft announced they were stopping their services at some point.

Reply:
Right.
How do you manage pictures in GitHub and Pelican?

Reply:
OK to change, but quite complex.